task_manager/api.py

```diff
@@ -14,20 +14,25 @@
 from .routes.users import model_fetch_user, model_get_update_user
 from .routes.users import model_post_update_user, model_delete_user
 
+from flask_restx import Resource, Namespace
```

Comment:
F811: redefinition of unused 'Resource' from line 6  <!-- Automatically posted by Lintly -->